pysma/__init__.py

```diff
@@ -154,7 +154,11 @@ def __len__(self):
 
     def __contains__(self, key):
         """Check if a sensor is defined."""
-        return key in self
+        try:
```

Comment:
Lets revert the change to this function

Reply:
OK, but this code will cause infinite recursions in my test. That's why I changed it to previous code. Maybe I did something wrong.

Reply:
Strange indeed. With your code, if the key exists, but the value of self[key] is falsy you will not return True or False, but None which is also falsy.